examples/01-Getting-started.ipynb

```diff
@@ -0,0 +1,1012 @@
+{
```

Comment:
<p>Can we add one more sentence/paragraph to it -&gt; why is NVTabular/preprocessing/feature engineering important and what are we doing in this example.</p><p> </p><p>Something like: Training a machine learning model requires often to preprocess data and engineer features that the algorithms will work. In our example, we want to train a neural networks with embedding layers based on two categorical features <code>user_id</code> and <code>movie_id</code> . Embedding layers require that the categorical features are continuous integers. In this example, we will show how to use Categorify Op to transform the categorical features for training a model   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/NVTabular/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>paragraph added</p>